packages/react/package.json

```diff
@@ -9,7 +9,7 @@
   "esnext": "./index.js",
   "types": "./index.d.ts",
   "sideEffects": [
-    "base.min-*"
+    "base.*"
```

Comment:
I'm a bit worried this is too broad rule, as this will affect files named "base.js", "base.component.tsx"

Could be just target the `base.module.css`? I gues it was `base.min-*` because the filename changed between builds?